app/controllers/document_images_controller.rb

```diff
@@ -132,6 +132,17 @@ def destroy
     end
   end
 
+  def download
+    image = Image.find(params[:image_id])
+    variant = image.crop_variant("960x640!").processed
```

Comment:
I think you can just use `image.crop_variant` here, as the width/height are defaulted in the model :-).

Reply:
There is an added explanation mark `960x640!` here which the default doesn't have. This forces the image to crop at the exact dimensions specified, without this the image can be a few pixels out due to ImageMagik trying to not distort the image on resizing.

I'm now curious as to whether we want to set the default to a force crop e.g add a explanation mark?

Reply:
Ooh, nice spot! Maybe it's worth adding a specific method on the model to explain this?

Reply:
Hmmm yep, when i uploaded this image and then downloaded it without the bang it was 1 pixel out at 960x639 which obviously isn't suitable as whitehall errors that the image isn't the correct size. 
![tree_png3498](https://user-images.githubusercontent.com/24479188/49862747-c09b8580-fdf6-11e8-848a-61a73b4970ff.png)

I've had a brief look to find the discussion i found about this on the web but can't seem to find it -.-

Reply:
Ah yeah sure, that makes sense that occasionally it could be a pixel off. Hmm I wonder whether we should drop a pixel normally or just distort ever so slightly :-/ but this step seems necessary for Whitehall so this is good.

Reply:
I don't think distorting the image by a pixel is going to affect the visual appearance of the image to the naked eye, so if it's required that the images are the exact dimensions they are cropped with then i think setting it to the default is okay?

However, if this needs further discussion i'll just use the bang with the download crop for now

Reply:
Yeah I'm not sure beyond Whitehall if these need to be the exact dimensions or not. I think if we roll with it being exact for this Whitehall upload but imagemagick defaults elsewhere, and then we'll see whether that causes any pain.